src/main/resources/com/google/api/codegen/ruby/sample.snip

```diff
@@ -142,14 +147,45 @@
   @end
 @end
 
+@private methodCall(class)
```

Comment:
Might be able to incorporate `paramList` into this as well, so long as it's called `methodCall`

```
@private methodCall(class)
  @if class.isPageStreaming
    @if class.methodCallFieldVarNames
      {@class.serviceVarName}.{@callChain(class.methodNameComponents)}({@paramList(class.methodCallFieldVarNames)}, {@class.pageStreaming.pageTokenName}: token)
    @else
      {@class.serviceVarName}.{@callChain(class.methodNameComponents)}({@class.pageStreaming.pageTokenName}: token)
    @end
  @else
    {@class.serviceVarName}.{@callChain(class.methodNameComponents)}({@paramList(class.methodCallFieldVarNames)})
  @end
@end
```